src/ranged.cpp

```diff
@@ -1573,6 +1573,17 @@ dispersion_sources player::get_weapon_dispersion( const item &obj ) const
         dispersion.add_range( std::max( vol - get_skill_level( skill_driving ), 1 ) * 20 );
     }
 
+    if( get_option< bool >( "IMPROVED_RANGED_ACCURACY" ) )
+    {
+        double cbmBonus =  has_bionic( bionic_id( "bio_targeting" ) ) ? 2 : 0;// insalled cbm work as bonus skill levels
+        double perSkillMult = 0.0833;// = 0.25/3 to get one bio_targeting effect every 3 levels of avgSkill
+        double avgSkill = ( double( get_skill_level( skill_gun ) + double( get_skill_level(
+                                     obj.gun_skill() ) ) ) ) / 2;
```

Comment:
Redundant external parentheses.

Also (just in case if I understand you correctly), do you mean a) `double( X + double ( Y ) ) / 2`, 
or b) `double( X + Y ) / 2`?